src/main/resources/parsePlayUrl.js

```diff
@@ -0,0 +1,710 @@
+// parsePlayUrl
+
+var parsePlayUrlVersion = 16;
+var parsePlayUrlBuild = "mmf";
+
+function parsePlayUrl(urlString) {
+	var url = urlString;
+	try {
+		url = new URL(urlString);
+	}
+	catch(error) {
+		console.log("Can't read URL: " + error);
+		return null;
+	}
+
+	var queryParams = {};
+	for (var queryItem of url.searchParams) {
+		queryParams[queryItem[0]] = queryItem[1];
+	}
+
+	return parseForPlayApp(url.hostname, url.pathname, queryParams, url.hash);
+}
+
+function parseForPlayApp(hostname, pathname, queryParams, anchor) {
```

Comment:
Is this function intended to wrap the entire Internet ?

Reply:
Half the the internet is enough.
For sure, we should refactor it.